RadarSDK/RadarSettings.m

```diff
@@ -133,22 +133,18 @@ + (void)setTrackingOptions:(RadarTrackingOptions *)options {
     [[NSUserDefaults standardUserDefaults] setObject:optionsDict forKey:kTrackingOptions];
 }
 
-+ (void)revertToFallbackTrackingOptions {
-    RadarTrackingOptions *_Nullable fallbackOptions = [self fallbackTrackingOptions];
-    if (fallbackOptions != nil) {
-        [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:[NSString stringWithFormat:@"Reverting to fallback options = %@", fallbackOptions]];
-        [[NSUserDefaults standardUserDefaults] setObject:[fallbackOptions dictionaryValue] forKey:kTrackingOptions];
-    }
++ (void)removeRemoteTrackingOptions {
```

Comment:
it doesn't look like we clear the object for the remote tracking options key on Android